package.json

```diff
@@ -40,5 +40,8 @@
     "hooks": {
       "pre-commit": "npx run format:staged"
     }
+  },
+  "dependencies": {
+    "lodash": "https://github.com/lodash/lodash.git#npm"
```

Comment:
This shouldn't be here. Uninstall it, and install it in the package where it belong :)

Reply:
yeah think this happened when i was having issues with getting the correct lodash ill clean that up now

Reply:
fixed